staging/src/k8s.io/apiserver/pkg/endpoints/handlers/workflow_test.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handlers
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestWorkflowID(t *testing.T) {
+	tests := []struct {
+		userAgent   string
+		subResource string
+		verb        string
+		contentType string
+		wanted      string
+	}{
+		{
+			wanted: "default",
```

Comment:
I wanted an error, how can a call not specify a verb?

Reply:
It shouldn't, I'm not excited to have to handle errors if you specified an empty string. I think it's fine as it is.